modules/aggs-matrix-stats/src/test/resources/rest-api-spec/test/stats/10_basic.yml

```diff
@@ -1,6 +1,9 @@
 # Integration tests for Matrix Aggs Plugin
 #
 "Matrix stats aggs loaded":
+    - skip:
+          reason: "contains is a newly added assertion"
```

Comment:
I believe the reason can be skipped when you specify a feature. it is required only when specifying versions

Reply:
@atorok ping not sure you have seen this comment

Reply:
Sorry @javanna I did not interpret that as something you would like me to change. 
The reason is overly repetitive even if used with version, but  it does seem to add some more context with someone not as familiar with this syntax. That's why I left it in.

Reply:
it's not a big deal, just that I think skip features contains is already self-explaining and adding a reason does not add value, but I am fine with keeping it. I did not mean that I wanted you to change it, I was ok with you replying that you are leaving it as-is, just so I know you've seen the comment ;)